unittest/python/crocoddyl_legacy/integrated_action.py

```diff
@@ -1,10 +1,11 @@
 import numpy as np
+import crocoddyl
```

Comment:
Do not create this folder (i.e. `crocoddyl_legacy`) and file here.
Put the Python integrator inside here: https://github.com/loco-3d/crocoddyl/blob/master/bindings/python/crocoddyl/utils/__init__.py#L390

Furthermore, I am not sure why we have an Euler integrator as well. This is included in the file that I have just shared.

Reply:
For the moment, I am keeping all the Python classes used to check the c++ code. Note that this file is not installed, only built internally for the unittest purpose.

Reply:
done